docs/rules/consistent-test-it.md

```diff
@@ -1,14 +1,26 @@
 # Have control over `test` and `it` usages (`consistent-test-it`)
 
+💼 This rule is enabled in the following
+[configs](https://github.com/jest-community/eslint-plugin-jest#shareable-configurations):
```

Comment:
can we link to the `README.md` file? Otherwise the link is broken on mobile

Reply:
Good point, updated.

Reply:
The link works fine for me on mobile, as-is.